tests/test_lightning.py

```diff
@@ -0,0 +1,7 @@
+from perceiver.model.lightning import DecoderConfig, EncoderConfig, LitImageClassifier
+
+
+def test_lit_image_classifier():
+    lit = LitImageClassifier((64, 64, 3), 2, 16, 16, EncoderConfig(), DecoderConfig(), optimizer_init={})
+
+    assert lit is not None
```

Comment:
I think having just 

```python
LitImageClassifier((64, 64, 3), 2, 16, 16, EncoderConfig(), DecoderConfig(), optimizer_init={})
```

should be sufficient as `lit` can only be `None` if instantiation fails which lets the test fail anyway.

Reply:
What about renaming `test_lightning.py` to `lightning_test.py`?